### PR TITLE
New Feature: Allow control how many lines to scroll at once using the mouse wheel on TextInput

### PR DESCRIPTION
**Pull request by DexerBR**
Currently vertical scrolling is limited to one line per scroll, and does not have an exposed option to control it. This PR allows you to scroll more than one line using a mouse wheel.
The number of rows scrolled at a time will default to 3.

</br>

Maintainer merge checklist
* [x] Title is descriptive/clear for inclusion in release notes.
* [x] Applied a `Component: xxx` label.
* [ ] Applied the `api-deprecation` or `api-break` label.
* [ ] Applied the `release-highlight` label to be highlighted in release notes.
* [ ] Added to the milestone version it was merged into.
* [ ] **Unittests** are included in PR.
* [x] Properly documented, including `versionadded`, `versionchanged` as needed.

### CODE REVIEW

**Comment by tshirtman** [comment]
#7615 was merged :)

**Comment by DexerBR** [comment]
I changed the number of lines that scroll by default to 3, as it is more convenient, but this can be discussed.

**Review by tshirtman — DISMISSED** [review]
LGTM

tests are failing because of expecting the (now) wrong value, should be easy to fix.

**Comment by misl6** [comment]
@DexerBR can you take care of the failing tests?

**Comment by DexerBR** [comment]
@misl6 I haven't had any experience with UI testing yet, but I can try to solve it next week. But if someone else is available to fix the tests, it would be very welcome!

**Comment by DexerBR** [comment]
How can I run these tests locally? Any documentation?

File: kivy/uix/textinput.py [hunk 1]
```diff
@@ -3821,6 +3824,16 @@ def _get_min_height(self):
     defaults to 0.
     '''
 
+    lines_to_scroll = BoundedNumericProperty(3, min=1)
+    '''Set how many lines will be scrolled at once when using the mouse scroll
+    wheel
```
Comment (pythonic64) [inline kivy/uix/textinput.py:3829]: A dot is missing at the end of the sentence.

**Comment by DexerBR** [comment]
@misl6 Tests have been fixed!

**Comment by DexerBR** [comment]
Hi @misl6 !
Is there anything else that needs to be addressed before merging this one?

**Review by misl6 — APPROVED** [review]
LGTM. And thank you for the heads up!

**Pull request merged**